caps/models.py

```diff
@@ -367,7 +426,7 @@ def get_available_name(self, name, max_length):
 overwrite_storage = OverwriteStorage()
 
 
-class PlanDocument(models.Model):
+class PlanDocument(CapsModel):
```

Comment:
do we need to replace the base model on every class? it looks like we only need it on the DataPoint class.

I don't suppose it does any harm but I think I have a slight preference for inheriting from standard classes unless there's a reason not to.

Reply:
Sure thing.